.github/workflows/buildandpublish.yml

```
@@ -0,0 +1,32 @@
+name: Build and publish Ansible
+on:
+  push:
+    tags:
+      - v*
+jobs:
+  build-and-publish:
+    name: Build and publish Ansible to PyPI
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
```

`master` is deprecated and is pointing to a year-old commit. Use the latest floating tag, at least:
```suggestion
    - uses: actions/checkout@v3
```
(or use a commit sha for the latest)